### ISSUE TITLE
When selecting (symbolic) links, show the target in the status bar

### ISSUE THREAD

**Issue by warpkanal** [marta-file-manager/marta-issues#233]
Would be helpful to know where a given symlink points to, e.g. by showing the target in the status bar once the symlink is selected in the filelist.

**Comment by yanex** [comment]
I don't think it should be a status bar message but some additional tooltip showing the original file path.

**Comment by warpkanal** [comment]
Where-ever you see fit for such information (i.e. NimbleCommander is showing it in the status bar).
Would however prefer a _mouseless_ location.
